xml/System/String.xml

```diff
@@ -4466,10 +4466,18 @@ Depending on Why you want to call the `Copy` method, there are a number of alter
         <Parameter Name="value" Type="System.Char" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The character to compare to the character at the end of this instance.</param>
+        <summary>Determines whether the end of this string instance matches the specified character.</summary>
```

Comment:
>acte [](start = 94, length = 4)

would be good if you can specify the "value" parameter.

Reply:
> Creates a new string with specific length and allow initialize the string after creating it through the action callback.
> Feel free to rephrase it.

Thank you, @tarekgh.

I actually found the PR where this method was merged (not sure why I couldn't find it before):
https://github.com/dotnet/coreclr/pull/13849/files
And here is the PR with the unit tests, so I could understand how it worked:
https://github.com/dotnet/corefx/pull/23872/files

Reply:
Do you mean like this?:
```suggestion
        <summary>Determines whether the end of this string instance matches the specified <paramref name="value" /> character.</summary>
```